model/Plan.js

```diff
@@ -11,7 +11,13 @@ const planSchema = new Schema({
   year_ids: [{ type: Schema.Types.ObjectId, ref: "Year" }],
   distribution_ids: [{ type: Schema.Types.ObjectId, ref: "Distribution" }],
   user_id: { type: String, required: true },
-  reviewers: { type: [String], default: [] },
+  reviewers: [
+    {
+      user_id: { type: String, ref: "User" },
+      pending: { type: Boolean },
```

Comment:
I think this should be changed to status, which is an enum of PENDING, ACCPTED, APPROVED, etc. This pending var alone does not cover enough state

Reply:
This has been noted and will be taken a look at in a later iteration.